docs/running/cwl.rst

```diff
@@ -225,6 +204,6 @@ Understanding toil log files:
 The new log file will be named something like:
 ::
 
-    file:---home-johnsoni-pipeline_1.1.14-ACCESS--Pipeline-cwl_tools-marianas-ProcessLoopUMIFastq.cwl_I-O-jobfGsQQw000.log
+    file:<path to cwl tool>.cwl_<job ID>.log
```

Comment:
Can you keep both versions? It is nice to see a real filename here

Reply:
👍